### ISSUE TITLE
Investigate flagging of unsure transcriptions

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeBMD2#519]
Compare FreeBMD1

**Comment by richardofsussex** [comment]
BMD1 uses bold face to indicate that there are at least two identical transcriptions of an entry. Non-bold text indicates a single transcription. Italic text for the District indicates either that the District name is an alternative or mis-spelled version of the name, or that the volume and/or page number does not match the expected values for that District at that time. Examples:

![image](https://user-images.githubusercontent.com/2197102/196715940-97a2a80c-08db-4381-a165-0f3b9745e14e.png)
Here the two transcriptions have different page numbers, and one of those numbers is out of range for that District in Sep 1837. This presentation makes it pretty clear that the first entry is more likely to be the correct one.

![image](https://user-images.githubusercontent.com/2197102/196716551-1b3190d0-a3d3-48b4-a4d3-0a89578e7959.png)
Here the District name isn't an approved or alternate form of name. The page which is the target of the link makes this plain:
![image](https://user-images.githubusercontent.com/2197102/196716938-cb42ba1e-3aef-4515-bbb9-92c53c15e098.png)
Our current display makes none of these distinctions, and in fact hides the volume/page information so that you have to look at the detail to even begin to guess which might be correct:
![image](https://user-images.githubusercontent.com/2197102/196717581-8fb12911-6a62-45d1-9476-9d4bc7828533.png)

**Comment by AlOneill** [comment]
Please remember that typographical differences such as bold or italic text are inaccessible to a screen reader user. We could use them as a starting point but we will need something else PDQ.

**Comment by richardofsussex** [comment]
Fair comment.  Is it possible to use the title attribute for this purpose? It would in my view be an improvement on BMD1 if users didn't have to follow a link to find out what the issue is with the District. The title could begin "Warning: " where there is an issue with the District or volume/page.

**Comment by AlOneill** [comment]
No, 'fraid not! Title attributes are inaccessible to those who cannot or do not use a pointing device. Forget the misinformation that hails the title attribute as a boon to accessibility — it is a pernicious myth.

**Comment by richardofsussex** [comment]
OK: you'll have to educate me as to how we deal with such situations.

Another point about the BMD1 design is that the District link goes to an intermediate page, specific to that entry, from which there are onwards links both to the UKBMD site and to the BMD1 information about that District. The BMD2 link goes straight to UKBMD and offers no way of accessing the BMD information about the District.

**Comment by richardofsussex** [comment]
There is room in the Entry Information page to include the summary information about the District which currently sits on the separate intermediate page:
![image](https://user-images.githubusercontent.com/2197102/196727498-e0511ec9-3ead-4c0e-89d8-c8832e4fc1b0.png)
This would allow us to add warnings as text, and would mean that we can keep the current links to UKBMD as they are.

**Comment by richardofsussex** [comment]
However, see #520 where I come to a different conclusion (!)

**Comment by richardofsussex** [comment]
The Districts table tells us the start and end quarters, so we can check that it did exist in the quarter specified in the entry. It also records the volume number for each major span of years (1837-51; 1852-1945; etc.) so that can be checked from that table. We can also check whether the authorized name for the District was used, or a synonym. (This may require use of the DistrictSynonyms and/or DistrictPseudonyms tables: the core Districts table also includes 'invented' names.)

In order to check whether the page number is likely to be correct (the actual issue in this example), we need a new source of information. As far as I can tell from the BMD1 source code, the expected page ranges for a particular district, year quarter and event type are deduced from a suitably-named CSV file.

**Comment by richardofsussex** [comment]
The bestguess table has a "Volume" index, on the combination of Volume, Page and QuarterNumber:
![image](https://user-images.githubusercontent.com/2197102/197174148-20802057-94b8-45f1-97b9-41b9b7bddd8c.png)
We could check how many entries there are for the combination of volume, page and quarternumber in an entry, and see if it corresponds to the number of entries you might expect in a valid page. If there is only one entry (i.e. the current one), it is almost certain to be an outlier, and can be flagged as such.

**Comment by richardofsussex** [comment]
In general, I think that we should be more specific in our comments on doubtful data. For example, the doubt about the second Eliza Greenwood entry above relates to the page number, yet it is the District name which is italicized. There is room in the Entry Information page to do this, and it will make the page more accessible - arguably the uncertainty information on the BMD1 pages isn't accessible at all, from what @AlOneill says - in which case whatever we achieve will be a major improvement.

**Comment by DeniseColbert** [comment]
Another for the steering group
